action.yml

```diff
@@ -82,13 +88,15 @@ runs:
             --target "${{ inputs.target }}" \
             --archive-name "${{ inputs.archive-name }}" \
             --changes-file "${{ inputs.changes-file }}" \
-            --extra-files "${{ inputs.extra-files }}"
+            --extra-files "${{ inputs.extra-files }}" \
+            --working-directory "${{ inputs.working-directory }}"
     - name: Generate SHA-256 checksum file (*nix)
       shell: bash
       run: |
         set -e
         set -x
 
+        cd ${{ inputs.working-directory || '.' }}
```

Comment:
There's no need for `|| '.'` everywhere since that's the default for this parameter.

Reply:
That's a good point, never made a GH action script so I was going for the "better safe than sorry". Noted tho.